.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## PR summary
+<!-- please include a brief summary of the changes in this PR -->
+
+
+## PR Checklist
+
+### General checklist
+Please make sure that your PR fulfills the following requirements:  
+- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] Dependencies have been updated as needed
+- [ ] .secrets.baseline updated as needed?
+
+#### Checklist for adding a new validation rule:
+- [ ] Added new validation rule definition (packages/ruleset/src/rules/*.js, index.js)
+- [ ] Added new validation rule implementation (packages/ruleset/src/functions/*.js, updated index.js)
```

Comment:
We should add an "if necessary" or something similar to this line since a function is not always required - many simple things can be achieved within the rule definition